### DISCUSSION TITLE
Welcome to VideoKit Discussions!

### DISCUSSION THREAD

**Discussion by olokobayusuf** [videokit-ai/videokit#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Summerflowerling** [comment]
Hi, When I tried to install VideoKit, I encountered error warning. 
1. First, the VideoKit has been installed follow the instruciton from README
![1709707603824](https://github.com/natmlx/videokit/assets/29397311/e33d881a-c078-41b1-8365-61c249724160)

2. Then I got error message like tihs:
![1709707614260](https://github.com/natmlx/videokit/assets/29397311/6daa4bbc-bb3e-4f0b-a48b-2bb9ac82dc8f)

3. I also can't find the VideoKit listed in the menu in project setting as well
![1709707648104](https://github.com/natmlx/videokit/assets/29397311/9808a8b0-2384-466b-be5d-a47224168108)

Any idea what might be missing? Thank you

**Reply by olokobayusuf** [reply]
Hey @Summerflowerling what version of Unity are you in? Can you share a screenshot of your Project Settings > Player Settings > Other Settings tab in Unity?

**Comment by Summerflowerling** [comment]
Hi @olokobayusuf 
I am using Unity 2020.3.27f1.
As for the screenshots are below:
![2fa9a3196ee2132af45093469331f2c](https://github.com/natmlx/videokit/assets/29397311/0f977d22-c820-4609-86d0-6dc3e5992618)
![d2a7bfcfaf462f5a71beed19b4c2dd5](https://github.com/natmlx/videokit/assets/29397311/19be71f8-a908-4e29-81ed-85e14e67fafe)
![28f19dea5a63f9852638c31bbd53eaf](https://github.com/natmlx/videokit/assets/29397311/26298a2a-48f6-40a8-b190-6e3ee46d1d60)
Thank you

**Reply by olokobayusuf** [reply]
Oh, you're on Unity 2020. VideoKit requires Unity 2022 LTS.
